ern-composite-gen/package.json

```diff
@@ -35,6 +35,7 @@
   },
   "homepage": "http://www.electrode.io",
   "dependencies": {
+    "cli-table": "^0.3.1",
```

Comment:
Any `yarn.lock` changes related to that?

Reply:
No this dependency and version range is already used in some other `ern-*` packages.